code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -298,6 +298,23 @@
 	icon_state = "grappabottle"
 	list_reagents = list("grappa" = 100)
 
+/obj/item/reagent_containers/food/drinks/bottle/sake
+	name = "Ryo's traditional sake"
+	desc = "Sweet as can be, and burns like fire going down."
+	icon_state = "sakebottle"
+	list_reagents = list("sake" = 100)
+
+/obj/item/reagent_containers/food/drinks/bottle/sake/Initialize()
+	. = ..()
+	if(prob(10))
+		name = "Fluffy Tail Sake"
```

Comment:
fuckup of the year

Reply:
my waifu is pure

Reply:
That game is about as pure as Kevin's and me combined